docs/content/author-bundles.md

```diff
@@ -142,7 +140,7 @@ parameters:
 
 Porter supports passing a file as a parameter to a bundle.
 
-For instance, a bundle might declare a parameter mytar of type file, located at /root/mytar when the bundle is run:
+For instance, a bundle might declare a parameter `mytar` of type `file`, located at `/root/mytar` when the bundle is run:
```

Comment:
You wouldn't know this since it's very inconsistent in the docs, but we are trying to move away from using inline code blocks. It renders funny and distracts the reader.

For example, this is hard to read and the blocks throw off the flow of the text:

![Screen Shot 2021-05-25 at 10 41 08 AM](https://user-images.githubusercontent.com/1368985/119527236-c1fe6580-bd45-11eb-9a23-2010a9b5be15.png)

If you'd like to do a sweep for that, and have us switch to some other formatting, which could mean none at all for most things, that would be helpful but we should sync up first on what we want it to look like.